docs/apps/components/sub-form.md

```diff
@@ -0,0 +1,148 @@
+          * ```
+            this.addClassName('table_n7rs950l', 'abc')
+            ```
+
+    2. `this.removeClassName(fields, className)`
+        ```js
+        this.removeClassName('table_n7rs950l', 'abc')
+        ```
+
+    3. `this.hide(fields)`
+        ```js
+        var fields= ['table_n7rs950l']
+        this.hide(fields)
+        ```
+    
+    4. `this.show(fields)`
+        ```js
+        var fields= ['table_n7rs950l']
+        this.show(fields)
+        ```
+    
+    5. `this.disable(['fields'])`
+        ```js
+        this.disable(['table_n7rs950l'])
+        ```
+    
+    6. `this.enable(['fields'])`
+        ```js
+        this.enable(['table_n7rs950l'])
+        ```
+
+    7.  `this.getComponent('component_ID')`
+        ```js
+        var subname = this.getComponent('table_n7rs950l');
+        console.log('getComponent', subname);
+        ```
+
+!!! info Add multiple classes
+    Use the following syntax to add multiple classes to a component:
+    ```js
+    this.addClassName('componentID', 'class1', 'class2')
+    ```
+
+## Config
+
+| **Name**|**Description**|**Icon**|
+|---------|---------------|--------|
+|**ID**| The sub-form component's unique identifier|<img src= "/apps/components/img/input_id.png">|
+|**Label Width**|Width of the label associated with an input field. It determines the horizontal space occupied by the label text|<img src= "/apps/components/img/input_labelwidth1.png">|
+|**Hide Label**| Hides the label on the form|<img src= "/apps/components/img/input_hidelabel.png">|
+|**Text Prompt**| A description to aid the user when completing the field|<img src= "/apps/components/img/input_textprompt.png">|
+|**Paging**|Enables pagination|<img src= "/apps/components/img/subform_paging.png">|
+|**Page Size**| Displays the number of entries per page|<img src= "/apps/components/img/subform_pagesize.png">|
+|**Show Control**|Displays control button|<img src= "/apps/components/img/subform_showcontrol.png">|
+|**Virtual Table**| Display elements inside the sub-form|<img src= "/apps/components/img/subform_virtualtable.png">|
+|**Default Value**| The default value which is filled in the field before the user changes it|<img src= "/apps/components/img/input_defaultvalue.png">|
+|**Custom Class**| An HTML class attribute which allows further customisation [See Form Attribute > Style Sheets](https://bani-appsection--connexcs-docs.netlify.app/apps/page-builder/#form-attribute)|<img src= "/apps/components/img/input_customclass.png">|
+|**Attribute Action**|Enable **Data Binding** to connect the data to UI. <br> Enable **Hidden** action to hide the field. <br> Enable **Disabled** action to make the field unusable. <br>Enable **Deletable** to delete the row in the sub-form. <br> Enable **Addible** to add a row in the subform. <br>Enable **Show word count** action make the word count visible|<img src= "/apps/components/img/input_attributeaction.png">|
```

Comment:
Remove: show word count and replace the image to accurately depict the attribute Actions available for sub form

Reply:
done